app/helpers/films_helper.rb

```diff
@@ -0,0 +1,11 @@
+require 'csv'
+
+module FilmsHelper
+  def get_csv_data
+    data = []
+    CSV.foreach("lib/assets/netflix_titles.csv", headers: true) do |row|
```

Comment:
also convert `"lib/assets/netflix_titles.csv"` into a constant variable in this file